boards/matek/h743/default.px4board

```diff
@@ -1,9 +1,9 @@
 CONFIG_BOARD_TOOLCHAIN="arm-none-eabi"
 CONFIG_BOARD_ARCHITECTURE="cortex-m7"
 CONFIG_BOARD_SERIAL_GPS1="/dev/ttyS1"
-CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS2"
+CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS5"
```

Comment:
The purpose of these labels is to map to what's physically on the board (what a user would see). On a typical pixhawk there are ports labelled telem 1, GPS, etc. 

Should we name these to match the actual documentation from Matek?

Reply:
I think we should use the manufacturer intended definition for that port, though I understand `telem1` and `telem2` are predefined by native pixhawks.

Reply:
> I think we should use the manufacturer intended definition for that port, though I understand `telem1` and `telem2` are predefined by native pixhawks.

Agreed! Can you make those changes in the board or do we need to add new serial port names? https://github.com/PX4/PX4-Autopilot/blob/90789be68f9d6c3c74597652b7a2681f4e3d2c28/Tools/serial/generate_config.py#L39-L84

Reply:
I cant change on the board itself since it writes TX7 and RX7 but on the website they mentioned `telem1` (see picture below) for in both INAV and Ardupilot mapping
If we want we can open up `URT7` in serial mapping, but I would prefer to leave it as it is
![Screenshot 2022-05-07 00:01:52](https://user-images.githubusercontent.com/64059887/167170107-8cda75e8-674a-4321-9910-a741492a22f4.png)

Reply:
Alright, whichever way you want to go is fine by me. Let me know if we need to add support for more custom serial port names.